packages/settings-library/src/settings_library/efs.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+
+from pydantic import Field
+
+from .base import BaseCustomSettings
+
+
+class AwsEfsSettings(BaseCustomSettings):
+    EFS_DNS_NAME: str = Field(
+        description="AWS Elastic File System DNS name",
+        example="fs-xxx.efs.us-east-1.amazonaws.com",
+    )
+    EFS_PROJECT_SPECIFIC_DATA_DIRECTORY: str = Field(default="project-specific-data")
+    EFS_MOUNTED_PATH: Path = Field(
+        default=Path("/data/efs"),
+        description="This is the path where EFS is mounted to the EC2 machine",
+    )
+    EFS_ENABLED_FOR_USERS: list[int] = Field(
+        description="This is temporary solution so we can enable it for specific users for testing purpose",
```

Comment:
Minor: Maybe make it friendlier to use by ingesting a list of mails instead of IDs. The User-IDs should in the long term become UUIDs I guess, so I would try not to cement them further

Reply:
At some time, this will likely reside in the `group_extra_properties` table. Currently, in the place where I intend to use it, I have only the user ID, which makes it convenient for my purposes.